Oracle/PCORNetLoader_ora.sql

```diff
@@ -961,36 +972,35 @@ left outer join
   on enctype.patient_num=v.patient_num and enctype.encounter_num=v.encounter_num;
 
 end PCORNetEncounter;
-/
-
-
-
-
-create or replace procedure PCORNetDiagnosis as

```

Comment:
I'm not sure what you're trying to fix by pulling the modifier temp table creation out of the stored procedures, but this won't work. Those temp tables (though it looks like Kun is creating full tables and dropping them before each run) need to be refreshed/created just before putting the data in the PopMedNet tables.